src/App.tsx

```diff
@@ -1,24 +1,22 @@
-import "./App.scss";
-import React, { useState } from 'react';
-import { AnimatePresence } from 'framer-motion';
-import WelcomeScreen from './views/WelcomeScreen/WelcomeScreen';
-import PaniniCreator from './views/PaniniCreator/PaniniCreator';
+import './App.scss'
+import React, { useState } from 'react'
+import { AnimatePresence } from 'framer-motion'
+import WelcomeScreen from './views/WelcomeScreen/WelcomeScreen'
+import PaniniCreator from './views/PaniniCreator/PaniniCreator'
 
 const App: React.FC = () => {
-  const [currentScreen, setCurrentScreen] = useState<'welcome' | 'panini' | 'success'>('welcome');
+	const [currentScreen, setCurrentScreen] = useState<'welcome' | 'panini' | 'success'>('welcome')
```

Comment:
**nitpick:** Let's make these values an enum, so you dont have to duplicate strings, eg. `enum Screens { 
  welcome,
  panini,
  success
}`